src/goto-instrument/nondet_static.h

```diff
@@ -20,6 +20,8 @@ Date: November 2011
 #ifndef CPROVER_GOTO_INSTRUMENT_NONDET_STATIC_H
 #define CPROVER_GOTO_INSTRUMENT_NONDET_STATIC_H
 
+#include <util/options.h>
```

Comment:
you can use a forward declaration instead if you use `std::list<std::string>` instead of `optionst::value_listt` below

Reply:
Unless there is a plan to drop the `optionst::value_listt` in the long term, I personally think using explicitly this type makes it clearer what is expected as a parameter.

Reply:
I personally disagree that it is clearer (the name value_listt carries strictly less information than list<string>), but I know people can have a different opinion on that so I won't insist.